models/public/colorization-v2-pytorch/accuracy-check.yml

```diff
@@ -0,0 +1,24 @@
+evaluations:
+  - name: colorization-v2-pytorch
+    module: custom_evaluators.colorization_evaluator.ColorizationEvaluator
+    module_config:
+      network_info:
+        colorization_network: {}
+
+        verification_network:
+          adapter: classification
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: imagenet_1000_classes
+          reader: pillow_imread
+
+          preprocessing:
+            - type: resize
+              size: 256
+              use_pillow: True
+              interpolation: Bicubic
+            - type: normalization
+              std: 255
```

Comment:
should not be normalization is a part of mo conversion?

Reply:
Normalization should not be is a part of mo conversion, because normalization is necessary for conversion RGB to Lab (because using numpy) as preprocessing, but part of mo conversion add normalization as part of model (so if use normalization in mo, normalization will be executed after RGB to Lab and will haven't effect).  Just in case I checked it and my assumptions were confirmed. But it might be useful to add `rgb_lab` as preprocessing in OMZ. What do you think about it?